CONTRIBUTING.md

```diff
@@ -91,22 +95,29 @@ When you upgrade `godo` you have to re-generate the mocks.
     make mocks
     ```
 
-### Build Scripts
+#### Build Scripts
+
+If you modify the build scripts, you can use `make shellcheck` to
+check your changes. You'll need to install [shellcheck](https://github.com/koalaman/shellcheck)
+to do so. Alternatively, you can open a "WIP" (Work In Progress) pull request
+and let TravisCI run shellcheck for you.
 
-If you modify the build scripts, use `make shellcheck` to check your changes.
 
 ## Releasing
 
-### Setup
+### Prerequisites
 
-To release `doctl`, you need to install:
+* [goreleaser](https://goreleaser.com/install/)
 
-* [gothub](https://github.com/itchio/gothub)
+* [docker](https://docs.docker.com/install/)
 
-And make it available in your `PATH`. You can use `go get -u` and add your
-`$GOPATH/bin` to your `PATH` so your scripts will find it.
+* a valid `GITHUB_TOKEN` environment variable with access to the
+  `digitalocean/doctl` repo. You can generate a token
+  [here](https://github.com/settings/tokens), it needs the `public_repo`
+  access.
 
-You will also need a valid `GITHUB_TOKEN` environment variable with access to the `digitalocean/doctl` repo. You can generate a token [here](https://github.com/settings/tokens), it needs the `public_repo` access.
+* a valid dockerhub login with access to the `digitalocean` account. Post
+  in #it_support to request access.
```

Comment:
> Post in #in_support to request access.

We might want to add a note that this is for DO employees only, just to avoid confusion for any public contributors.